src/libmongoc/src/mongoc/mongoc-handshake.h

```diff
@@ -28,64 +28,7 @@ BSON_BEGIN_DECLS
 
 #define MONGOC_HANDSHAKE_APPNAME_MAX 128
 
-/**
- * mongoc_handshake_data_append:
```

Comment:
Ah, I now realize much of the RST text comes from the comment here.

Reply:
Right, there wasn't a lot else to go on!